examples/glob/src/main.rs

```diff
@@ -13,40 +13,58 @@ fn main() {
     let mut settings = Config::default();
     settings
         // File::with_name(..) is shorthand for File::from(Path::new(..))
-        .merge(File::with_name("conf/00-default.toml")).unwrap()
-        .merge(File::from(Path::new("conf/05-some.yml"))).unwrap()
-        .merge(File::from(Path::new("conf/99-extra.json"))).unwrap();
+        .merge(File::with_name("conf/00-default.toml"))
+        .unwrap()
+        .merge(File::from(Path::new("conf/01.json")))
+        .unwrap()
+        .merge(File::from(Path::new("conf/99-extra.toml")))
+        .unwrap();
 
     // Print out our settings (as a HashMap)
-    println!("\n{:?} \n\n-----------",
-             settings.try_into::<HashMap<String, String>>().unwrap());
+    println!(
+        "\n{:?} \n\n-----------",
+        settings
+            .try_into::<HashMap<String, config::Value>>()
+            .unwrap()
+    );
```

Comment:
This is partly a formatting change, please include only the relevant bits in this PR.